pkg/lib/cockpit.d.ts

```diff
@@ -199,4 +199,6 @@ declare module 'cockpit' {
 
     function format_bytes(n: number): string;
     function format(format_string: string, ...args: unknown[]): string;
+
+    function message(arg: string | BasicError): string
```

Comment:
Looks like we also test cases where it can be a `{problem: string}` or a `{message: string}`... `test-chan.js` has a whole lot of fun examples...

I love how everything always needs to be so complicated...

Reply:
I think those test cases are ought to be fixed then, what do you think?